### ISSUE TITLE
Typogrify makes changes which should be accepted

### ISSUE THREAD

**Issue by drgrigg** [pcrooker/edgar-rice-burroughs_tarzan-of-the-apes#6]
These are mostly hairspaces associated with dashes, or non-breaking spaces with names.

Just run typogrify again and commit the changes.

**Comment by pcrooker** [comment]
Hi David, I did run typogriphy before the last commit. Do you have a couple of examples?

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by drgrigg** [comment]
Here's a screenshot of my GitHub desktop after running typogrify on your repository:

![Screenshot 2021-12-18 13-50-28](https://user-images.githubusercontent.com/7020970/146626564-5b69855c-c7d5-4eea-b54a-497bbe3a8459.jpg)

**Comment by drgrigg** [comment]
I've been messing about with a hex editor because it's not easy to figure out what's going on with git diff, and it looks like the change typogrify wants to make is a reversal of the order of the em-dash and the word joiner character. In any case, typogrify is usually correct, so I'd accept the changes it wants to make.

**Comment by pcrooker** [comment]
I think if must have ran se typogrify in a different repository - I had mispelled burroughs and kept the misspelled repo - I should have just deleted the bloody thing. Sorry, will fix.

Yes, the word join character is a pain. I've been using visual studio code with a modified 'Gremlins tracker for visual studio' module that highlights nominated characters in different colours. I added the various UTF chars to those highlighed by the module. I can just make out the word joiner chars - the thin line in orange, the green colours are nbsp and em dashes:

[image.png]

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
